datastore/shared/services/environment_service.py

```diff
@@ -37,3 +37,16 @@ def ensure_cache(self, name: str) -> None:
     def is_dev_mode(self) -> bool:
         value = self.try_get(DATASTORE_DEV_MODE_ENVIRONMENT_VAR)
         return value is not None and value.lower() in ("1", "on", "yes", "true")
+
+    def get_from_file(self, name: str) -> str:
+        if name not in self.cache:
+            path = os.environ.get(name + "_FILE", None)
+            if path is not None:
+                with open(path) as file_:
+                    self.cache[name] = file_.read()
+            else:
+                if self.is_dev_mode():
+                    self.cache[name] = os.environ.get(name, None)
+                else:
+                    raise EnvironmentVariableMissing(name + "_FILE")
```

Comment:
I would expect the `DATASTORE_DATABASE_PASSWORD` var to still work in production mode.
Although when setting up OS with the manage-service it will probably never get used, someone might for some reasons still find it convenient to provide a password directly.

This would leave the user with the option to make their setup less secure, so it can be argued to leave this option as is and and force the more secure method, however I still think this is unexpected behavior.

Reply:
I think, we should provide a secure way to set the `DATASTORE_DATABASE_PASSWORD` and `DATASTORE_DATABASE_USER`. So in prod the _FILE variant should be used.

Reply:
I suggest to discuss this "feature", because we can then do it consistently for all repos.

Reply:
@GabrielInTheWorld wants to open an issue about this.

Reply:
See #157